src/commands/cargo.ts

```diff
@@ -1,6 +1,23 @@
 import * as io from '@actions/io';
 import * as core from '@actions/core';
 import * as exec from '@actions/exec';
+import * as cache from '@actions/cache';
+import * as http from '@actions/http-client';
+import * as path from 'path';
+
+export async function resolveVersion(crate: string): Promise<string> {
+    const url = `https://crates.io/api/v1/crates/${crate}`;
+    const client = new http.HttpClient(
+        '@actions-rs (https://github.com/actions-rs/)',
+    );
+
+    const resp: any = await client.getJson(url);
```

Comment:
`eslint` complains about this `any`, but without it the `resp.result` fails and I don't know which type should be used here instead...

Reply:
Oh, the CI eslint seems to be fine with that, so nvm